terraform/modules/query-service-infra/sqs.tf

```diff
@@ -0,0 +1,54 @@
+
+resource "aws_sqs_queue" "query_queue" {
```

Comment:
To parameterize your lambdas, this file would be the place to declare an output of queue name, arn, or url and feed it through to the lambda environment. Then you are assured that everything is locked together.

Reply:
👍